pygmt/src/solar.py

```diff
@@ -0,0 +1,91 @@
+"""
+solar - Plot day-night terminators.
+"""
+
+from pygmt.clib import Session
+from pygmt.helpers import (
+    build_arg_string,
+    fmt_docstring,
+    kwargs_to_strings,
+    use_alias,
+)
+
+
+@fmt_docstring
+@use_alias(
+    R="region",
+    J="projection",
+    G="fill",
+    B="frame",
+    Td="day_night",
+    Tc="civil_twilight",
+    Tn="nautical_twilight",
+    Ta="astronomical_twilight",
+    W="pen",
+    U="timestamp",
+    V="verbose",
+    X="xshift",
+    Y="yshift",
+    p="perspective",
+)
+@kwargs_to_strings(R="sequence", p="sequence")
+def solar(self, **kwargs):
+    r"""
+    A plotting function the day-night terminator. Additionally, it can
```

Comment:
>  A plotting function the day-night terminator.

This sentence doesn't read well.

Reply:
I probably changed what I wanted to say and never went back to correct it; whoops! Updated in `
bf0e2d5`.